src/main/java/org/json/NumberConversionUtil.java

```diff
@@ -0,0 +1,142 @@
+package org.json;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+public class NumberConversionUtil {
```

Comment:
Let's not make this public. We may wish to refactor this class in the future. For now make it private or default (package) visible.

Reply:
Static methods meant to be used by functional classes being marked private do not add up. But yes, @stleary - unnecessary making these classes open for used by clients might not be a great idea, there are plenty of scope of improvements in there. So marked them default visibility.

Reply:
That is correct. I simply meant that the class itself should be default or private, not the static methods.

Reply:
However, I now recall that private outer classes is not a thing in Java. I was thinking inner class scopes. Default visibility is correct here.

Reply:
@stleary @johnjaylward - can this PR be accepted please?

Reply:
@rudrajyotib Will get to it as soon as I can after the work of #811 is completed.